.env

```diff
@@ -0,0 +1,2 @@
+
+password=pk.e1f8f92e635d436ee387f6d6afb1d612
```

Comment:
Good work on getting started Emily 👍 

This file was committed and needs to be .gitignored. Check your `.gitignore` and ensure that `.env` file is added.

You may need to go ahead and remove the file entirely and make a commit. Then add `.env` to .gitignore then commit again. Finally you can add back in your key to `.env` file after it has been properly ignored.

Grab a TA when ya can to help ya here:)